projects/canopy/src/lib/brand-icon/brand-icon.component.ts

```diff
@@ -23,6 +23,8 @@ export type BrandIconSize = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
   templateUrl: './brand-icon.component.html',
   styleUrls: [ './brand-icon.component.scss' ],
   encapsulation: ViewEncapsulation.None,
+  standalone: true,
+  host: { ngSkipHydration: 'true' },
```

Comment:
Was this added because you specifically noticed issues with hydration on this component?

Reply:
yes, there are a few places where hydration was an issue on Dotcom. I have tested this option and it seams to be working fine, no changes in the functionality. 
Also I think this is only used when enabling SSR so it will only do it's thing on SSR apps like Dotcom